dev/wlp-gradle/subprojects/repos.gradle

```diff
@@ -30,16 +30,16 @@ repositories {
             }
         }
     } else {
-        mavenCentral()
-         maven {
-            url ("https://oss.sonatype.org/content/repositories/snapshots/")
+        maven {
+            url ("http://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo/")
             metadataSources {
                 mavenPom()
                 artifact()
             }
         }
+        mavenCentral()
         maven {
-            url ("http://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo/")
+            url ("https://oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
I thought that @cbridgha was going to be removing this snapshots repo?

Reply:
PR #15161 just updated this so now it causes a merge conflict I am sure.